### ISSUE TITLE
Recovery Check opens perception check popup

### ISSUE THREAD

**Issue by hendryl** [Larkinabout/fvtt-token-action-hud-pf2e#132]
**Versions:**
- Foundry 315 Version 11 Stable on Windows 11
- Pathfinder Second Edition 5.13.6
- socketlib 1.0.13
- Token Action HUD Core 1.5.3
- Token Action HUD Pathfinder 2e 1.5.10
- No other modules enabled

**Steps to reproduce:**
1. Select a player character
2. Apply Dying condition
3. Hover on `Utility` tab
4. Select `Recovery Check`

**Expected:**
- Recovery check is rolled.

**Current:**
- Recovery check is rolled, and perception check popup is opened.

**Notes:**
- Works as expected if I click recovery check on the character sheet.

![image](https://github.com/Larkinabout/fvtt-token-action-hud-pf2e/assets/10694317/dbb249d7-bb9d-49f8-8019-98807626f9d3)

**Comment by Silvertower** [comment]
I actually noticed this in the code yesterday when I was looking for a different issue. I'll go back and check legacy code. Probably an unintentional regression.

**Comment by Silvertower** [comment]
Indeed, this is a regression and will be fixed in the next release.

**Comment by Silvertower** [comment]
This issue will be fixed in the next release.

**Comment by Silvertower** [comment]
Fixed in Token Action HUD PF2e version 1.5.11.
